presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/docker/DockerFiles.java

```diff
@@ -70,12 +70,12 @@ public synchronized String getDockerFilesHostPath()
         return dockerFilesHostPath.toString();
     }
 
-    public String getDockerFilesHostPath(String file)
+    public Path getDockerFilesHostPath(String file)
```

Comment:
should `public synchronized String getDockerFilesHostPath()` above also return Path?